peachjam/models/judgment.py

```diff
@@ -191,18 +191,14 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
 
-class JudgmentMediaSummaryFile(AttachmentAbstractModel):
-    SAVE_FOLDER = "media_summary_files"
+class AttachedFileNature(models.Model):
```

Comment:
This can move into `peachjam/models/core_document_model.py` since it's no longer judgment-specific.

Reply:
Ditto for AttachedFiles